modules/ROOT/pages/flex-gateway-delete.adoc

```diff
@@ -0,0 +1,40 @@
+= Delete a Flex Gateway
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-disabled.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
+image:logo-hybrid-active.png[xref="deployment-strategies.adoc#hybrid-deployments", title="Hybrid"]
+image:logo-server-disabled.png[xref="deployment-strategies.adoc#anypoint-platform-pce-deployments", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]
+
+To delete a gateway and its replicas in Runtime Manager:
+
+. From Anypoint Platform, select *Runtime Manager*.
+. Select *Flex Gateway* in the left menu.
+. Click the action menu button for the gateway you want to delete.
+. Click Delete gateway. +
+image:flex-gateway-delete.png[Delete Flex Gateway]
+
+If there are APIs associated to the gateway you want to delete, Runtime Manager displays a message prompting you to either:
+
+* Move the associated APIs to another gateway.
+* Delete all associated APIs.
+
+Once you have moved or deleted all the associated APIs, follow the steps previously described to delete a gateway and confirm the deletion.
+
+Deleted gateways are shown with status **Deleted** for 30 days before it is removed from the gateways list.
+
+Deleting a gateway from the Runtime Manager UI simply removes the connection between Anypoint Platform and the gateway and all its replicas.
+This action does not remove the gateway from the server. To manually delete a gateway from your server, see xref:gateway::flex-conn-manage.adoc[Managing Flex Gateway in Connected Mode]
```

Comment:
Can you elaborate what you mean by `does not remove the gateway from the server`

Reply:
Adressed! Removed the language.